projects/abseil-cpp/build.sh

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+#Compiling and building dependencies
+mkdir -p build
+cd build
+cmake -DCMAKE_CXX_COMPILER=$CXX -DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+	-DBUILD_SHARED_LIBS=OFF -G "Unix Makefiles" ..
+make
+#Building string_escape_fuzzer
+$CXX $CXXFLAGS -I../ \
+	$SRC/string_escape_fuzzer.cc -o $OUT/string_escape_fuzzer \
+	$LIB_FUZZING_ENGINE \
+	absl/strings/libabsl_strings.a \
```

Comment:
We can integrate with bazel as part of the upstream "ideal integration", where we merge these targets upstream. For now, this is OK as an initial integration. 

If we want to make things less brittle here, perhaps we can use the `find` command to find all the ".a" files and then merge them into one .a file (to avoid dependency order issues).

Reply:
Wouldn't it just be easier to write a CMakeLists.txt file? See https://github.com/abseil/abseil-hello/blob/master/cmake-hello/CMakeLists.txt for an example.

Reply:
@derekmauro I tried making a build system using bazel(not familiar with bazel that much), no issues in linking abseil-cpp but i ran into issues with the fuzzer coverage report something to do with envrionment variables being changed during bazel build, i am trying to debug it.